src/view/checkbox-list.js

```diff
@@ -5,7 +5,7 @@ const createCheckboxTypeList = (array) => {
     const lowerType = type.toLowerCase();
     return `<div class="event__type-item">
       <input id="event-type-${lowerType}-1" class="event__type-input  visually-hidden" type="radio" name="event-type" value="${lowerType}">
-      <label class="event__type-label  event__type-label--${lowerType}" for="event-type-${lowerType}-1">${type}</label>
+      <label class="event__type-label  event__type-label--${lowerType}" for="event-type-${lowerType}-1"  data-point-type="${type.toLowerCase()}">${type}</label>
```

Comment:
У тебя ж есть loverType.
```suggestion
      <label class="event__type-label  event__type-label--${lowerType}" for="event-type-${lowerType}-1"  data-point-type="${lowerType}">${type}</label>
```